### PR TITLE
Update color/theme dependencies

### PR DESCRIPTION
**Pull request by jvgreiner**
Fixes BLUI-4936.

#### Changes proposed in this Pull Request:
This is to test https://github.com/etn-ccis/blui-react-native-component-library/pull/436

- Update imports on Colors
- Update POD hermes disbaled
- Use @brightlayer-ui/colors": "4.0.0-alpha.1" & "@brightlayer-ui/react-native-themes": "7.0.0-alpha.0"

linting in showcase (kitchensink) dirty

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by joebochill** [comment]
Make sure you give your PRs a meaningful title...when I see this in a list of PRs I have no idea what it's meant to do / change / fix.

**Comment by ektaghag-eaton** [comment]
We need to update package @brightlayer-ui/react-native-components to display new components which we added in md3. 
Screenshot 2023-11-16 at 12 39 33 PM

**Comment by ektaghag-eaton** [comment]
This works perfectly fine from `blui-react-native-component-library`
Screenshot 2023-11-16 at 1 00 27 PM

**Comment by ektaghag-eaton** [comment]
Logged an issue for failing test cases https://github.com/etn-ccis/blui-react-native-showcase-demo/issues/109.

**Review by surajeaton — COMMENTED** [review]
The title of PR says ` remove old components demo`, but the changes are only related to `colors` and `themes` packages.

**Review by huayunh — APPROVED** [review]
Looks good.

Are we using the following files? If not, shall we remove them?
* router/navigation-drawer.tsx
* components/kitchen-sink.tsx
* components/user-menu-example.tsx

**Review by ektaghag-eaton — APPROVED** [review]

File: ios/Podfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ target 'BluiReactNativeShowcaseDemo' do
   use_react_native!(
     :path => config[:reactNativePath],
     # Hermes is now enabled by default. Disable by setting this flag to false.
-    :hermes_enabled => flags[:hermes_enabled],
+    :hermes_enabled => flags[false],
```
Comment (joebochill) [inline ios/Podfile]: Pretty sure there is no entry in the flags array called false
```suggestion
    :hermes_enabled => false,
```

**Pull request merged**